hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ContainerImporter.java

```diff
@@ -95,18 +95,17 @@ public void importContainer(long containerID, Path tarFilePath,
       throws IOException {
     if (!importContainerProgress.add(containerID)) {
       deleteFileQuietely(tarFilePath);
-      LOG.warn("Container import in progress with container Id {}",
-          containerID);
-      throw new StorageContainerException("Container " +
-          "import in progress with container Id " + containerID,
+      String log = "Container import in progress with container Id ";
+      LOG.warn(log + "{}", containerID);
+      throw new StorageContainerException(log + containerID,
```

Comment:
nit: no need for placeholders, since the message string is always created for the exception.

```suggestion
      String log = "Container import in progress with container Id " + containerID;
      LOG.warn(log);
      throw new StorageContainerException(log,
```